pkg/document/json/root_test.go

```diff
@@ -127,4 +127,39 @@ func TestRoot(t *testing.T) {
 		nodeLen = len(richText.Nodes())
 		assert.Equal(t, 1, nodeLen)
 	})
+
+	t.Run("garbage collection for container test", func(t *testing.T) {
+		root := testhelper.TestRoot()
+		ctx := testhelper.TextChangeContext(root)
+
+		newInt := func(n int) *json.Primitive {
+			return json.NewPrimitive(n, ctx.IssueTimeTicket())
+		}
```

Comment:
I think the benefit of function extraction is small. This function feels like an early abstraction. 🤔
What if we put it back inline?

Reply:
Thanks for the feedback! :D Your requested changes have been reflected.